### PR TITLE
Added PostgreSQL timestamptz type

### LINKED ISSUE #260: Can support 'timestamp with timezone' ?
I am using Postgresql and Timestamp with timezone columns.

Now I try phoenix `diff` command, return error.

```
❯ vendor/bin/phoenix diff --source local --target testing

In Column.php line 89:

  Type "timestamp with time zone" is not allowed. 

```

### PR DESCRIPTION
**Pull request by chatii**
fixed #260 

Added timestamp tz support and testing.
test result: https://github.com/chatii/phoenix/actions/runs/1154885299

### CODE REVIEW

File: src/Database/Element/Column.php [hunk 1]
```diff
@@ -28,6 +28,7 @@ class Column
     const TYPE_DATE = 'date';
     const TYPE_TIME = 'time';
     const TYPE_DATETIME = 'datetime';
+    const TYPE_DATETIME_TZ = 'timestamptz';
```
Comment (lulco) [inline src/Database/Element/Column.php]: If constant is called DATETIME, I would like the value be also datetime. Then just remap it in query builder / adapter

**Comment by lulco** [comment]
Thanks for contribution! Can you please update changelog and add new type to docs? https://github.com/lulco/phoenix/blob/master/docs/migrations/columns.md

File: src/Database/Element/Column.php (hunk 1 above)
Comment (lulco) [reply src/Database/Element/Column.php]: Or name the constant TYPE_TIMESTAMP_TZ

**Comment by chatii** [comment]
@lulco thanks for res!
const renamed `TYPE_TIMESTAMP_TZ`, and updated change log, column.md

**Comment by lulco** [comment]
thank you!

**Pull request merged**